### PR TITLE
Fix h264 end dec apps

### PR DESCRIPTION
**Pull request by jjomier**
Updated H264 support for Holoscan 1.0

### CODE REVIEW

File: dev_container [hunk 1]
```diff
@@ -475,6 +475,11 @@ launch() {
         conditional_opt+=" -v /opt/yuan/qcap/lib:/opt/yuan/qcap/lib"
     fi
 
+    # add tegra directory
+    if [ -d  /usr/lib/aarch64-linux-gnu/tegra ]; then
```
Comment (wendell-hom) [inline dev_container:479]: Which libraries are we depending on from here, are they not mounted with `runtime=nvidida`?

File: applications/orsi/lib/CMakeLists.txt [hunk 2]
```diff
@@ -19,21 +19,21 @@ project(orsi_app_lib CXX)
 find_package(holoscan 0.6 REQUIRED CONFIG
              PATHS "/opt/nvidia/holoscan" "/workspace/holoscan-sdk/install")
 
-add_library(${PROJECT_NAME}
+add_library(orsi_app_lib EXCLUDE_FROM_ALL
```
Comment (wendell-hom) [inline applications/orsi/lib/CMakeLists.txt:22]: With these changes, do the orsi lib no longer get compiled if we are not building the orsi app?

File: dev_container (hunk 1 above)
Comment (jjomier) [reply dev_container:479]: These are the multimedia libraries

File: applications/orsi/lib/CMakeLists.txt (hunk 2 above)
Comment (jjomier) [reply applications/orsi/lib/CMakeLists.txt:22]: This is correct. This was an issue that it was built by default so building the h264 will build the orsi lib.
Could be put in another MR but good to put here.

File: dev_container (hunk 1 above)
Comment (agirault) [reply dev_container:479]: Is this an issue on dGPU only? the nvidia runtime should handle things

Comment (jjomier) [reply dev_container:479]: the nvidia runtime does not add this directory

File: applications/orsi/lib/CMakeLists.txt (hunk 2 above)
Comment (wendell-hom) [reply applications/orsi/lib/CMakeLists.txt:22]: fine with having this here

**Review by wendell-hom — APPROVED** [review]

**Pull request merged**